src/include/pg_version_compat.h

```diff
@@ -137,23 +136,6 @@ RelationGetSmgr(Relation rel)
 #define SetListCellPtr(a, b) ((a)->ptr_value = (b))
 #define RangeTableEntryFromNSItem(a) ((a)->p_rte)
 #define QueryCompletionCompat QueryCompletion
-#else /* pre PG13 */
-#define lnext_compat(l, r) lnext(r)
-#define list_delete_cell_compat(l, c, p) list_delete_cell(l, c, p)
-#define pg_plan_query_compat(p, q, c, b) pg_plan_query(p, c, b)
-#define planner_compat(p, c, b) planner(p, c, b)
-#define standard_planner_compat(a, c, d) standard_planner(a, c, d)
-#define CMDTAG_SELECT_COMPAT "SELECT"
-#define GetSequencesOwnedByRelation(a) getOwnedSequences(a, InvalidAttrNumber)
-#define GetSequencesOwnedByColumn(a, b) getOwnedSequences(a, b)
-#define ExplainOnePlanCompat(a, b, c, d, e, f, g, h) ExplainOnePlan(a, b, c, d, e, f, g)
-#define SetListCellPtr(a, b) ((a)->data.ptr_value = (b))
-#define RangeTableEntryFromNSItem(a) (a)
-#define QueryCompletionCompat char
-#define varattnosyn varoattno
-#define varnosyn varnoold
-#endif
-#if PG_VERSION_NUM >= PG_VERSION_12
```

Comment:
shouldn't we remove in between all the things starting `#if PG_VERSION_NUM >= PG_VERSION_12` and ending `endif`?

Reply:
I only removed the if condition since we want those lines if the pg version is bigger than 12, and given that pg version is going to be 13 or higher the if condition is no longer necessary. Actually, this if condition could have been removed when dropping pg11 support.

Reply:
I'm confused by this file. For example, why do we have `lnext_compat /list_delete_cell_compat  ` etc in `version_compat.h` that is the same for all versions?

I guess that deserves another PR to clean-up to adjust these.